internal/k8s/selfregister.go

```diff
@@ -21,25 +21,29 @@ package k8s
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"os"
+	"strings"
 	"sync"
 
+	"github.com/networkservicemesh/cmd-admission-webhook/internal/config"
```

Comment:
Why is this import needed?

Reply:
@bharath-avesha its just go fmt thing...